pkg/volume/iscsi/iscsi.go

```diff
@@ -198,6 +199,7 @@ type iscsiDisk struct {
 	chap_discovery bool
 	chap_session   bool
 	secret         map[string]string
+	InitiatorName  string
```

Comment:
Understand.
Since I didn't use InitiatorName in DetachDisk(), initiatorName should start with lowercase.